client/src/assets/images/global/video-lang.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
```

Comment:
What is the source of this file?

Also please cleanup the SVG file (remove the xml header and `id` because it could lead to some issues in our HTML)

Reply:
It is the 'video' icon from https://robbiepearce.com/softies/, which I modified manually :)

Reply:
:+1: